src/Apps/Collect2/Routes/Collection/Components/Header/DefaultHeader.tsx

```diff
@@ -72,8 +72,14 @@ export const CollectionDefaultHeader: FC<Props> = ({
                   })
                 }}
               >
+                {/**
+                 * Don't add 5px of padding to left margin of the first image
+                 * nor to the right marging of the last image in the header
+                 * because the header should bleed to the edge of the screen
+                 */}
                 <Image
-                  mx={0.5}
+                  ml={i === 0 ? 0 : 0.5}
```

Comment:
This feels like something that might be better handled with `&:first-child` and `&:last-child` selectors applied to the parent element (defined as a styled component).

[Example](https://github.com/artsy/reaction/blob/5a8c9dffbcd4cee4b0074f7af23b20188a184731/src/Components/Publishing/RelatedArticles/Canvas/RelatedArticlesCanvas.tsx#L104)